runtime/tools/spec-builder/package.json

```diff
@@ -18,6 +18,8 @@
     "@polkadot/types": "^1.26.1",
     "@polkadot/util": "^3.0.1",
     "as-scale-codec": "^0.1.1",
+    "lodash": "^4.17.19",
+    "mocha": "^8.0.1",
```

Comment:
normally the libraries that are part of the test infrastructure are saved as devDependencies